Students/Farcas Silviu Vlad/Assignment 1.2/Fibonacci.java

```diff
@@ -0,0 +1,16 @@
+public class Fibonacci {
+	public static void main(String[] args){
+		int i=0;
+		long f=1, sum=0;
+		for(i=0; f <= 4_000_000; i++){
+			if (f % 2 == 0) sum += f;
+			f = fib(i);
+		}
+		System.out.println("The sum is " + sum);
+	}
+	public static long fib(int n) {
```

Comment:
Very good. Can you tell me the complexity of the recursive approach to Fibonacci?